cmd/server/assets/codestatus/show.html

```diff
@@ -67,6 +74,15 @@ <h5 class="mb-1">Expiry</h5>
         // Disable the submit if already expired.
         $buttonInvalidate.prop('disabled', true);
       });
+
+      {{if .code.HasLongExpires}}
+      let $longCodeExpiresAt = $('#long-code-expires-at');
+      // Start countdown
+      countdown($longCodeExpiresAt, longExpires, function() {
```

Comment:
do-da-do---do, da-da-do-do-do, do-da-do---do...

Reply:
you ok?

also - I made this better

Reply:
It's really hard to put musical intonation into words. I was going for: https://www.youtube.com/watch?v=9jK-NcRmVcw

Reply:
https://www.youtube.com/watch?v=bagCdMUuevQ